docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -106,3 +110,44 @@ endpoint. Client key and certificate locations are shown by `tsh db config`.
 ![MongoDB Compass more options](../../../img/database-access/compass-more-options@2x.png)
 
 Click on the "Connect" button.
+
+## DBeaver
+
+[DBeaver](https://dbeaver.io/) is a SQL client software application and database administration tool.
+
+### MySQL 
+
+<Admonition type="note">
+  Teleport's DBeaver MySQL integration only supports MySQL server 8.0.2 or older due to changes made in required handshake parameters starting with MySQL 8.0.3 and above.
```

Comment:
add this to config.json variable to avoid hardcodes and refer to it with `(=mysql.dbeaver_min_ver=)`